client/blocks/express-checkout/express-checkout.js

```diff
@@ -0,0 +1,28 @@
+/* global wc_stripe_payment_request_params */
+
+import React from 'react';
+import { Elements, ExpressCheckoutElement } from '@stripe/react-stripe-js';
+
+export const ExpressCheckout = ( { stripe } ) => {
+	const options = {
```

Comment:
Are these hardcoded options intended?

Reply:
Yes. This is just for this initial PR. We will replace it with real values in the later PRs.